packages/microservices/server/server-tcp.ts

```diff
@@ -40,7 +40,7 @@ export class ServerTCP extends Server implements CustomTransportStrategy {
 
   constructor(private readonly options: TcpOptions['options']) {
     super();
-    this.port = this.getOptionsProp(options, 'port') || TCP_DEFAULT_PORT;
+    this.port = this.getOptionsProp(options, 'port', TCP_DEFAULT_PORT);
     this.host = this.getOptionsProp(options, 'host') || TCP_DEFAULT_HOST;
     this.socketClass =
       this.getOptionsProp(options, 'socketClass') || JsonSocket;
```

Comment:
Why not do the same for all of these assignments? (+1 line below I couldn't include here)
```suggestion
    this.port = this.getOptionsProp(options, 'port', TCP_DEFAULT_PORT);
    this.host = this.getOptionsProp(options, 'host', TCP_DEFAULT_HOST);
    this.socketClass =
      this.getOptionsProp(options, 'socketClass', JsonSocket);
```